flink-cdc-connect/flink-cdc-source-connectors/flink-connector-sqlserver-cdc/src/test/java/org/apache/flink/cdc/connectors/sqlserver/testutils/SqlserverCompareUuidTest.java

```diff
@@ -0,0 +1,58 @@
+package org.apache.flink.cdc.connectors.sqlserver.testutils;
+
+import org.apache.flink.cdc.connectors.base.utils.ObjectUtils;
+import org.apache.flink.cdc.connectors.sqlserver.source.utils.SqlServerUtils;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class SqlserverCompareUuidTest {
+    private static final Logger LOG = LoggerFactory.getLogger(SqlserverCompareUuidTest.class);
+    @Test
+    public void testWorkWithGuids() throws Exception {
```

Comment:
Please use `assertEquals` to verify UUID sorting results instead of logging them for manual checks.